drag_and_drop_v2/utils.py

```diff
@@ -5,3 +5,24 @@
 # Make '_' a no-op so we can scrape strings
 def _(text):
     return text
+
+
+def ngettext(text_singular, text_plural, n):
+    if n == 1:
+        return text_singular
+    else:
+        return text_plural
+
+
+class FeedbackMessages(object):
+    FINAL_ATTEMPT_TPL = _('Final attempt was used, highest score is {score}')
+    MISPLACED_ITEMS_RETURNED = _('Misplaced items were returned to item bank.')
+
+    CORRECTLY_PLACED_SINGULAR_TPL = _('Correctly placed {correct_count} item.')
+    CORRECTLY_PLACED_PLURAL_TPL = _('Correctly placed {correct_count} items.')
```

Comment:
@mtyaka Created a follow-up task https://tasks.opencraft.com/browse/OC-1831 and reworked so that ngettext are actually translated as late as possible, thorugh XBlock's `i18n` service.

Unfortunately, I was not able to make translations work in devstack (and looks like they aren't working in sandboxes too), so I really can't check if it actually helps. On the other hand, looks like entire XBlock translation thing is broken.